test/controllers/SameAccountControllerSpec.scala

```diff
@@ -34,15 +34,13 @@ import scala.concurrent.Future
 
 class SameAccountControllerSpec extends PlaySpec with GuiceOneServerPerSuite with MockitoSugar with BeforeAndAfterEach with AtedTestHelper {
 
-  val mockBCConnector: BusinessCustomerFrontendConnector = mock[BusinessCustomerFrontendConnector]
   val injview = app.injector.instanceOf[views.html.sameAccount]
```

Comment:
Can you rename this even though its not part of you ticket. A better self documenting name like injectedView would be good.